libraries/AP_HAL_ChibiOS/HAL_ChibiOS_Class.cpp

```diff
@@ -234,6 +233,7 @@ static void main_loop()
                                    pd.fault_thd_prio,
                                    pd.fault_icsr);
     }
+#endif
```

Comment:
Comments for what this `#endif` is closing (IE `#endif // HAL_NO_LOGGING`) is a lot better for long term readability, particularly if there isn't a single line in the middle.